pb_plugins/dcsdkgen.egg-info/PKG-INFO

```diff
@@ -0,0 +1,10 @@
+Metadata-Version: 1.0
+Name: dcsdkgen
+Version: 0.1a0
+Summary: Autogenerator for the DroneCore SDK Bindings
+Home-page: @TODO
+Author: @TODO
+Author-email: @TODO
+License: UNKNOWN
```

Comment:
BSD 3-clause.

Reply:
File should not be in git, added the whole folder to gitignore